src/main/kotlin/snyk/oss/OssBulkFileListener.kt

```diff
@@ -52,33 +57,46 @@ class OssBulkFileListener : SnykBulkFileListener() {
             if (event.file == null || !event.isFromSave) continue
             val file = event.file!!
             val activeProject = ProjectUtil.getActiveProject()
-            ProgressManager.getInstance().run(object : Task.Backgroundable(
-                activeProject,
-                "Snyk: forwarding save event to Language Server"
-            ) {
-                override fun run(indicator: ProgressIndicator) {
-                    val param = DidSaveTextDocumentParams(TextDocumentIdentifier(file.toLanguageServerURL()), file.readText())
-                    languageServer.textDocumentService.didSave(param)
-                }
-            })
+            ProgressManager.getInstance().run(
+                object : Task.Backgroundable(
+                    activeProject,
+                    "Snyk: forwarding save event to Language Server",
+                ) {
+                    override fun run(indicator: ProgressIndicator) {
+                        val param =
+                            DidSaveTextDocumentParams(
+                                TextDocumentIdentifier(file.toLanguageServerURL()),
+                                file.readText(),
+                            )
+                        languageServer.textDocumentService.didSave(param)
+                    }
+                },
+            )
         }
     }
 
-    private fun dropOssCacheIfNeeded(project: Project, virtualFilesAffected: Set<VirtualFile>) {
+    private fun dropOssCacheIfNeeded(
+        project: Project,
+        virtualFilesAffected: Set<VirtualFile>,
+    ) {
         val snykCachedResults = getSnykCachedResults(project)
         if (snykCachedResults?.currentOssResults != null) {
-            val buildFileChanged = virtualFilesAffected
-                .filter { scanInvalidatingFiles.contains(it.name) }
-                .find { ProjectRootManager.getInstance(project).fileIndex.isInContent(it) }
+            val buildFileChanged =
+                virtualFilesAffected
+                    .filter { scanInvalidatingFiles.contains(it.name) }
+                    .find { ProjectRootManager.getInstance(project).fileIndex.isInContent(it) }
             if (buildFileChanged != null) {
                 snykCachedResults.currentOssResults = null
                 log.debug("OSS cached results dropped due to changes in: $buildFileChanged")
             }
         }
     }
 
-    private fun updateCacheAndUI(filesAffected: Set<SnykFile>, project: Project) {
-        val cache = getSnykCachedResults(project)?.currentSnykCodeResultsLS ?: return
+    private fun updateCacheAndUI(
+        filesAffected: Set<SnykFile>,
+        project: Project,
+    ) {
+        val cache = getSnykCachedResults(project)?.currentOSSResultsLS ?: return
```

Comment:
This is the only non-formatting change, `currentSnykCodeResultsLS` to `currentOSSResultsLS` @bastiandoetsch